nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -702,6 +702,43 @@ in {
         '';
       };
     };
+
+    ensureUsers = mkOption {
+      default = {};
+      description = lib.mdDoc ''
+        List of user accounts which get automatically created if they don't
+        exist yet. This option does not delete accounts which are not listed
+        anymore.
+      '';
+      example = {
+        user1 = {
+          passwordFile = /secrets/user1-localhost;
```

Comment:
I think passwordFile options must be set with quoted strings, or else Nix will copy the secret into the world-readable Nix store at eval time (not so secret anymore).